plugins/pencilblue/controllers/admin/content/articles/article_form.js

```diff
@@ -44,23 +44,43 @@ module.exports = function(pb) {
             }
 
             self.article = results.article;
-            var tabs   = self.getTabs();
-
-            self.setPageName(self.article[pb.DAO.getIdField()] ? self.article.headline : self.ls.get('NEW_ARTICLE'));
-            self.ts.registerLocal('angular_script', '');
-            self.ts.registerLocal('angular_objects', new pb.TemplateValue(self.getAngularObjects(tabs, results), false));
-            self.ts.load('admin/content/articles/article_form', function(err, data) {
-                self.onTemplateRetrieved('' + data, function(err, data) {
-                    var result = '' + data;
-                    self.checkForFormRefill(result, function(newResult) {
-                        result = newResult;
-                        cb({content: result});
-                    });
-                });
-            });
+            if(!self.article.author) {
+              self.article.author = self.session.authentication.user._id.toString();
```

Comment:
It is uncomfortably verbose but we need to refer to the ```_id``` paramter via:

```
self.session.authentication.user[pb.DAO.getIdField()].toString()
```

This helps reduce churn to support multiple DBs in the future.